doc/cephfs/add-remove-mds.rst

```diff
@@ -61,8 +68,8 @@ means limiting its cache size.
 
 Adding an MDS
 =============
-
-#. Create an mds data point ``/var/lib/ceph/mds/ceph-${id}``. The daemon only uses this directory to store its keyring.
+   
```

Comment:
Any reason for white-space? (I guess it is not intended)

Reply:
I was trying to follow the style of formatting that I saw in the doc. If you see other headings too, there's a white space following them.

Reply:
I've removed the unwanted white space

Reply:
I've removed the unwanted white space